pyfar/constants/speed_of_sound.py

```diff
@@ -52,3 +53,95 @@ def speed_of_sound_simple(temperature):
     t_0 = pf.constants.absolute_zero_celsius
     c_ref = pf.constants.reference_speed_of_sound
     return c_ref*np.sqrt((temperature-t_0)/(t_ref-t_0))
+
+
+def speed_of_sound_ideal_gas(
+        temperature, relative_humidity, atmospheric_pressure=None,
+        saturation_vapor_pressure=None):
+    """Calculate speed of sound in air using the ideal gas law.
+
+    The speed of sound in air can be calculated based on chapter 6.3 in [#]_.
+    All input parameters must be broadcastable to the same shape.
+
+    Parameters
+    ----------
+    temperature : float, array_like
+        Temperature in degree Celsius.
+    relative_humidity : float, array_like
+        Relative humidity in the range of 0 to 1.
+    atmospheric_pressure : float, array_like, optional
+        Atmospheric pressure in pascal, by default
+        :py:attr:`reference_atmospheric_pressure`
+    saturation_vapor_pressure : float, array_like, optional
+        Saturation vapor pressure in Pa.
+        If not given, the function
+        :py:func:`~pyfar.constants.saturation_vapor_pressure` is used.
+        Note that the valid temperature range is therefore also dependent on
+        :py:func:`~pyfar.constants.saturation_vapor_pressure`.
```

Comment:
I checked the build docs and saw that the links don't work. Maybe it must be :py:attr:?

Reply:
Ive renamed the method, thats why, thanks for catchig